utxo/src/storage/view_impls.rs

```diff
@@ -24,21 +24,17 @@ mod utxosdb_utxosview_impls {
     use super::*;
 
     pub fn utxo<S: UtxosStorageRead>(db: &S, outpoint: &OutPoint) -> Option<Utxo> {
-        db.get_utxo(outpoint).unwrap_or_else(|e| {
-            panic!(
-                "Database error while attempting to retrieve utxo from the database: {}",
-                e
-            )
-        })
+        db.get_utxo(outpoint).expect("Database error while attempting to retrieve utxo")
```

Comment:
Why remove the extra info?

Reply:
It should be the same but simpler. As per documentation `expect` wll print the message from `Err`:

[[Panics](https://doc.rust-lang.org/std/result/enum.Result.html#panics)
Panics if the value is an [Err](https://doc.rust-lang.org/std/result/enum.Result.html#variant.Err), with a panic message including the passed message, and the content of the [Err] (https://doc.rust-lang.org/std/result/enum.Result.html#variant.Err).](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect)

Reply:
I see.